### PR TITLE
Fixes #13, Loading data to database

### LINKED ISSUE #13: Docker Container Initialization Error with louis-db
**Issue Description:**  
Error when trying to launch postgres.sh in louis-db-server.

**Steps to Reproduce:**  
1. Clone the `louis-db` repository.
2. Go to the branch `issue8-missing-chunks` to have access to the bin folder
3. Create a symbolic link to `~/src/louis-db/bin/postgres.sh` using the command: `ln -s ~/src/louis-db/bin/postgres.sh`.
4. Execute `./postgres.sh`.
5. Check the active Docker containers using `docker ps`.
6. Observe that the expected Docker container for louis-db is missing.
7. To investigate further, check the logs of the louis-db server with the command: `docker logs louis-db-server`.

**Error Message:**
```
chmod: /var/run/postgresql: Operation not permitted
Error: Database is uninitialized and superuser password is not specified.
       You must specify POSTGRES_PASSWORD to a non-empty value for the
       superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".

       You may also use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all
       connections without a password. This is *not* recommended.

       See PostgreSQL documentation about "trust":
       https://www.postgresql.org/docs/current/auth-trust.html
```
**Expected Behavior:**
The Docker container for louis-db should be created and initialized without errors.

**Operating System: Windows 10**
- Windows Subsystem for Linux (WSL) Version: 2
- WSL Distribution: Ubuntu
- Visual Studio Version: 1.82.0
- Docker Version: 24.0.5

### PR DESCRIPTION
**Pull request by JolanThomassin**
With this commit, I have enabled the capability to load dumps into the database, and it appears to be functioning correctly.

**Committed changed:**
- Introduced a **new environment variable**, `POSTGRES_PASSWORD`, which represents the database password. The `PGPASSWORD` variable now exclusively **handles the user password**.
- Included a **usage example** in `dump-versioned-schema.sh` for cases when an argument is missing.
- Revised `lib.sh` due to apparent issues with the `search_path` when attempting to use it.
- Added **comments** to `load-versioned-schema.sh` and completed **testing**, confirming that **the code appears to function correctly**.
- Modified `postgres.sh` to **replace the password environment** variable from `PGPASSWORD` to `POSTGRES_PASSWORD`.
- Created `setup-db-docker.sh`, which **ensures that the setup file functions correctly**, **checks for errors or missing variables**, and **verifies if the user or database already exists** to prevent errors or duplicates if applicable.

**Related issues:**
- [Missing Environment Variables in .env.template #16](https://github.com/ai-cfia/louis-db/issues/16)
- [Docker Container Initialization Error with louis-db #13](https://github.com/ai-cfia/louis-db/issues/13)

**Related pull-request:**
- [issue #8: install doc and missing_chunk_queries #12](https://github.com/ai-cfia/louis-db/pull/12)
- [Fixes #13, Loading data to database #19](https://github.com/ai-cfia/louis-db/pull/19)

### CODE REVIEW

File: bin/dump-versioned-schema.sh [hunk 1]
```diff
@@ -4,6 +4,7 @@ DIRNAME=`dirname $0`
 
 if [ -z $2 ]; then
     echo "usage: $0 source_schema output_schema"
+    echo "exemple: louis_v005 to louis_v006"
```
Comment (melanie-fressard) [inline bin/dump-versioned-schema.sh:7]: Example: $0 louis_v005 louis_v006

**Review by melanie-fressard — CHANGES_REQUESTED** [review]

File: bin/dump-versioned-schema.sh (hunk 1 above)
Comment (JolanThomassin) [reply bin/dump-versioned-schema.sh:7]: Indeed, I'll fix that, thank for noticing 👍

**Review by rngadam — APPROVED** [review]

File: DEVELOPER.md [hunk 2]
```diff
@@ -13,7 +13,9 @@
 
   * PGHOST: the hostname or IP address of the server where the PostgreSQL database is hosted.
 
-  * PGPASSWORD: the password for authentication when connecting to the PostgreSQL database.
+  * PGPASSWORD: the password for the user authentication when connecting to the PostgreSQL database.
+
+  * POSTGRES_PASSWORD: the password for the database, for authentication when connecting to the PostgreSQL database.
```
Comment (melanie-fressard) [inline DEVELOPER.md:18]: Add USER one too

**Review by melanie-fressard — APPROVED** [review]
👍🏻

**Pull request merged**